doc-content/shared-kie-docs/src/main/asciidoc/Install/testing-clustering-environment-proc.adoc

```diff
@@ -0,0 +1,122 @@
+[id='testing-clustering-environment-proc_{context}']
+= Testing that your infrastructure meets the minimum hardware and performance requirements for a viable high availability (HA) on-premises environment.
+
+When creating a production-ready high availability (HA) on-premises infrastructure for {CENTRAL}, you must ensure it meets the minimum hardware and performance requirements for a viable HA environment, and to mitigate performance related issues. This section describes the environment elements, an example environment, and how to test the performance.
+
+There are four main components; {CENTRAL}, the message system (AMQ), the indexing server ({DATAGRID}), and a shared file system (NFS/GlusterFS/Ceph).
+
```

Comment:
I suggest the following revised intro for this module. First sentence could be streamlined, I actually don't know what you're referring to in the second sentence, and the third sentence could be a little clearer I think. Also note that per IBM SG, we should avoid empty "there is/are". And needs to be "... components:" with a colon, not semi.

So something like this. Wdyt?

> When you create a production-ready high availability (HA) on-premises infrastructure for {CENTRAL}, you must ensure that it meets the minimum hardware and performance requirements for a viable HA environment. An HA on-premise infrastructure consists of the following four main components: {CENTRAL}, the message system (AMQ), the indexing server ({DATAGRID}), and a shared file system (NFS/GlusterFS/Ceph).

Reply:
I like the revised intro, thanks!